### ISSUE TITLE
Help Text Panel: Calanderization

### ISSUE THREAD

**Issue by rmroot** [ORNL-AMO/VERIFI#440]
Fill out help text panel for monthly meter data (calanderization-help component)

**Comment by koay9f** [comment]
![image.png](https://images.zenhubusercontent.com/5cd48a2af8cffa5a19122d27/208d4a41-e760-40ad-8406-601fa7ec8d45)
not actually sure where this will go so don't be too wordy

**Comment by koay9f** [comment]
might actually be:
![image.png](https://images.zenhubusercontent.com/5cd48a2af8cffa5a19122d27/6c9a65d2-61e0-4293-8841-44dd749614cc)

**Comment by JL102** [comment]
@pricecr Because the button to open the calendarization modal is now "Select Method", I changed `Select “View Calendarization Modal” or “Select Method”` to `Click “Select Method”`. Alternatively, we could make it `Use the "Select Method" button`.
